sov-modules/sov-modules-macros/src/dispatch/dispatch_query.rs

```diff
@@ -28,7 +28,7 @@ impl<'a> StructDef<'a> {
 
             quote::quote!(
                 #enum_ident::#name(message)=>{
-                    let #name = <#ty as sov_modules_api::ModuleInfo::#type_generics>::new(working_set.clone());
+                    let #name = <#ty as sov_modules_api::ModuleInfo>::new(working_set.clone());
```

Comment:
(Question): Why are we removing the generics here?

Reply:
`ModuleInfo` used to have generic type `ModuleInfo<C: Context>` but I changed it to the associated type, and now we don't need the generic parameter anymore.